General.ruleset

```diff
@@ -0,0 +1,115 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
SA1408 and S1067 seem to be very similar. Which one would you recommend?

Reply:
I played a bit with if statements to brake these rules. And these rules are not similar, they check different code issues. So I would leave both these rules enabled.

Reply:
Yeah, my mistake.